fd6130/hsl-bundle/1.0/config/packages/fd_hsl.yaml

```diff
@@ -0,0 +1,3 @@
+fd_hsl:
+    paginator:
+        default_limit: 30
```

Comment:
This looks like default values. 

Flex is smart enough to install this bundle without a recipe.

Reply:
Ok, done removing the hsl-bundle.